### ISSUE TITLE
Login buttons load slow on Teach

### ISSUE THREAD

**Issue by iamjessklein** [mozilla/id.webmaker.org#257]
It took about 30 seconds before the Log In | Signup buttons loaded in the upper left hand corner of the page. It looked as though nothing was loading while I waited. This might be a good opportunity to provide the user with some feedback.

**Comment by alicoding** [comment]
I don't think this is directly a bug on id.webmaker.org @iamjessklein. It could be teach-api bug or just how teach site doing the call to id.wb.org I'm not sure.

cc @toolness

**Comment by toolness** [comment]
I believe this was because teach-api.herokuapp.com is running on a free 1x web dyno that was [asleep](https://devcenter.heroku.com/articles/dynos#dyno-sleeping), and you logging into teach woke it up. This won't generally be the case in production.

That said, if the user's internet is slow, a similar effect will happen. I assume that while the buttons were loading, the area looked as described in https://github.com/mozilla/teach.webmaker.org/pull/626:

<img src="https://cloud.githubusercontent.com/assets/124687/7077719/2818afd2-dee1-11e4-9096-697cf4d9b300.jpg">

That *is* the feedback :)  We could potentially add something akin to [Font Awesome animations](http://fortawesome.github.io/Font-Awesome/examples/) to add a throbber/spinner effect, but aside from that or changing the text, there isn't much we can do pre-v1.

**Comment by secretrobotron** [comment]
@toolness want this moved over to the teach repo?

**Comment by toolness** [comment]
Sure!
